includes/abbreviations.md

```diff
@@ -0,0 +1,33 @@
+<!-- Material for MkDocs shows a tooltip when you hover over any abbreviation in this list. -->
```

Comment:
I'm still thinking about this recommendation:

> If everyone knows an abbreviation, use it without explanation

I've basically added all abbreviations I could find into the list. But maybe we should drop really common abbreviations. But what's common depends on the previous experience of the reader... 😄 

What kind of assumptions are we making about the readers of our documentation? Do we aim for:

- Seasoned developers
- People new to programming, but who want to use a dependency bot
- Team managers who review/approve new bots before developers can use/install them
- Somebody else I'm missing to list

Also there's something to be said for being consistent, and just shoving all abbreviations in the list... 😄 

What do you think?

Reply:
I think we can assume a reasonable maturity. For example they know API and JSON